### PR TITLE
Improve quick start

### PR DESCRIPTION
**Pull request by Murderlon**

### CODE REVIEW

File: docs/quick-start.mdx [hunk 1]
```diff
@@ -13,58 +13,46 @@ Uppy is a sleek, modular JavaScript file uploader that integrates seamlessly
...
 
 :::
 
 <QuickStartLinks
 	items={[
 		{
-			name: 'Examples',
-			description: 'Try out one of our extensive example projects',
-			link: 'https://github.com/transloadit/uppy/tree/main/examples',
+			name: 'I want a full featured, extendable dashboard',
```
Comment (arturi) [inline docs/quick-start.mdx:28]:
```suggestion
			name: 'I want a full featured, extendable UI with drag-drop, file previews and upload progress',
```

File: docs/quick-start.mdx [hunk 2]
```diff
@@ -13,58 +13,46 @@ Uppy is a sleek, modular JavaScript file uploader that integrates seamlessly
...
-			name: 'Uploaders',
+			name: 'Which uploader do I need?',
 			description: 'Choosing the uploader you need',
 			link: '/docs/guides/choosing-uploader',
 		},
 		{
-			name: 'Dashboard',
-			description: 'Powerful, responsive, and pluggable UI',
-			link: '/docs/dashboard',
+			name: 'I want to download from remote sources',
```
Comment (arturi) [inline docs/quick-start.mdx:38]:
```suggestion
			name: 'I want to add files from remote sources like Google Drive or Dropbox',
```

File: docs/quick-start.mdx (hunk 1 above)
Comment (Murderlon) [reply docs/quick-start.mdx:28]: I need to keep the sentences short or it doesn't look as good / less readable

Screenshot 2023-03-22 at 13 57 08

**Comment by arturi** [comment]
Maybe we drop “I want”. But people on Quick Start probably don't know what Dashboard is? And what is a remote source, why are you downloading from it?

**Comment by arturi** [comment]
:+1:

**Pull request merged**